### ISSUE TITLE
[Wii] Sonic Mania Wii not loading from USB when loaded from forwarder channels

### ISSUE THREAD

**Issue by saulfabregwiivc** [Mefiresu/RSDKv5-Decompilation#22]
As the title states, this Wii port doesn't load any resources from USB storage if launched from a Wii forwarder channel.
(It loads fine from USB if launched from Homebrew Channel).

Also it loads fine if loaded from SD, from both HBC and Wii forwarder channel.

Any fix for this?

PD: I will post the forwarder channel i've used
[Sonic Mania - S0MN.zip](https://github.com/Mefiresu/RSDKv5-Decompilation/files/10970978/Sonic.Mania.-.S0MN.zip)
.

**Comment by Deotex2** [comment]
Says modloader is loaded by default but dont know how to add them to this port can someone please tell me how to add the no decorations special stage mod the release tells you to add. That discord server is no help.

**Comment by deepfriedfilth** [comment]
@Deotex2 I found [these instructions](https://gbatemp.net/threads/sonic-mania-for-wii.619890/post-10137466) on GBAtemp thread

**Comment by deepfriedfilth** [comment]
@saulfabregwiivc possibly due to this? https://github.com/Mefiresu/RSDKv5-Decompilation/issues/15#issuecomment-1293802802

**Comment by Mefiresu** [comment]
> @saulfabregwiivc possibly due to this? [#15 (comment)](https://github.com/Mefiresu/RSDKv5-Decompilation/issues/15#issuecomment-1293802802)

No, things changed a lot since then. Since v0.5.0, the engine now uses a custom path given in `meta.xml`. Forwarders will need to be compatible with it in order to work.
